### ISSUE TITLE
Vertical Waveforms in Late Night

### ISSUE THREAD

**Issue by tlibra10** [mixxxdj/mixxx#13976]
### Feature Description

It would be nice to have the choice of a vertical waveform.
Everything is good with mixxx, but a vertical wavform would be a good idea.
Plus not to forget about the "Crossfader" auto re-center, when switching from auto mix to manual.

I am currently using the unofficial release of 2.6 which I consider the best version. 
So if the features were exact to 2.6, with a vertical waveform, it would be great.

**Comment by Swiftb0y** [comment]
Vertical waveforms are in theory supported, the skin just needs use them. What skin do you use?

**Comment by tlibra10** [comment]
I use "Late Night" skin in version 2.6

**Comment by Swiftb0y** [comment]
CC @ronso0 wdyt? Is it feasible to add vertical scrolling waveforms in a useful way to Late Night?

**Comment by tlibra10** [comment]
I have seen it on other displays that DJ's use and I think that it looks good. Takes up less room but gives the same information.
It's mainly just a thought. 
At the end of the day, both formats does exactly the same job. 
I just thought it looks better and quite professional.
That's just my opinion. Like I said, Just a thought. Thank you.

**Comment by ronso0** [comment]
As already responded in https://mixxx.discourse.group/t/an-idea-for-the-next-update/30816/2?u=ronso
none of the official skins' layout is designed for vertical waveforms, all assume landscape screens.

TL;DR
Yes, might work, but definitely quite some work to cram this into the current design.
I won't do it : \
A new skin design build around vertical waveforms is the better approach IMHO
____
Current LateNight might be modded to have vertical waveforms:
a) right or left of the main skin container, or
b) simply where they are now with some spacers or (resizable) splitters at both sides, which obviously only makes sense for portrait screens

**Comment by ronso0** [comment]
Let's keep this open so we can track it.
Vertical waveforms are a valid request, but it's probably not going to happen in LateNight so I changed the title.

**Comment by tlibra10** [comment]
![image](https://github.com/user-attachments/assets/1f8bebad-c5f2-4f45-84aa-f14908773e50)

**Comment by m0dB** [comment]
FWIW, the accelerated waveforms all should support vertical display (after all, it’s just a matter of rotating the matrix passed to the shader. I tested this with a quick&dirty hack of Latenight at some point.

And note to self: I should also do this for the upcoming rendergraph implementation.
